recipes-core/images/iot2050-image-base.bb

```diff
@@ -12,7 +12,7 @@ inherit image
 
 DESCRIPTION = "IOT2050 Debian Base Image"
 
-IMAGE_INSTALL += "u-boot-iot2050-config"
+IMAGE_INSTALL += "${@('u-boot-' + d.getVar('MACHINE')) if d.getVar('QEMU_IMAGE') == '1' else 'u-boot-${MACHINE}-config' }"
```

Comment:
u-boot-*-config is needed for interacting with the U-Boot env. I don't think we have a persistent env with U-Boot under QEMU. In any case, we don't need u-boot-<machine> in that case.

Reply:
A new yml file is created for compiling u-boot qemu version. Removed from image.